agent/agent.go

```diff
@@ -295,15 +295,19 @@ func (a *Agent) prepareNextTask(ctx context.Context, nextTask *apimodels.NextTas
 
 func shouldRunSetupGroup(nextTask *apimodels.NextTaskResponse, tc *taskContext) bool {
 	// we didn't run setup group yet
+	setupGroup := false
+	var msg string
 	if !tc.ranSetupGroup {
-		return true
+		msg = "running setup group because we haven't yet"
```

Comment:
are you intentionally replacing the message each time?

Reply:
no 😭